src/notify.js

```diff
@@ -35,7 +35,7 @@ function stopBarking() {
 }
 
 async function showMeetingAlert(evt, line, givingUpAfter) {
-  const title = evt.summary + ' ' + evt.startDate,
+  const title = `Late No More: ${evt.summary} ${evt.startDate}`,
```

Comment:
this doesn't display the way I want it to. I want Late No More to be the dialog title but it's showing a blank dialog title currently.

Also is there a way to include an icon? 

![221227-Late-No-More-Icon-4](https://user-images.githubusercontent.com/1034890/209779279-7b23e061-c5af-4129-9d51-1e674a0818e8.jpg)